docs/samples/line/hook.md

```diff
@@ -0,0 +1,123 @@
+# Outside of chart
+
+```js chart-editor
+// <block:setup:2>
+const DATA_COUNT = 8;
+const MIN = 10;
+const MAX = 100;
+
+Utils.srand(8);
+
+const labels = [];
+for (let i = 0; i < DATA_COUNT; ++i) {
+  labels.push('' + i);
+}
+
+const numberCfg = {count: DATA_COUNT, min: MIN, max: MAX};
+
+const data = {
+  labels: labels,
+  datasets: [{
+    data: Utils.numbers(numberCfg)
+  }]
+};
+// </block:setup>
+
+// <block:annotation:1>
+const annotation = {
+  type: 'line',
+  borderColor: 'black',
+  borderWidth: 3,
+  scaleID: 'y',
+  value: 55,
+  beforeDraw: (ctx) => drawExtraLine(ctx)
+};
+// </block:annotation>
+
+// <block:utils:3>
+function drawExtraLine(context) {
+  const ctx = context.chart.ctx;
+  const {x, y, x2, y2, options} = context.element;
+  ctx.save();
+  ctx.lineWidth = options.borderWidth;
+  ctx.strokeStyle = options.borderColor;
+  ctx.setLineDash([6, 6]);
+  ctx.lineDashOffset = options.borderDashOffset;
+  // Draw
+  ctx.beginPath();
+  ctx.moveTo(x - 40, y);
+  ctx.lineTo(x, y);
+  ctx.stroke();
+  ctx.beginPath();
+  ctx.moveTo(x2, y2);
+  ctx.lineTo(x2 + 40, y);
+  ctx.stroke();
+  ctx.restore();
+  return true;
+}
+// </block:utils>
+
+/* <block:config:0> */
+const config = {
+  type: 'line',
+  data,
+  options: {
+    layout: {
+      padding: {
+        right: 50
+      }
+    },
+    scales: {
+      y: {
+        stacked: true
+      }
+    },
+    plugins: {
+      annotation: {
+        clip: false,
+        annotations: {
+          annotation
+        }
+      }
+    }
+  }
+};
+/* </block:config> */
+
+var actions = [
```

Comment:
Same as in previous actions block, const instead of var

Reply:
@LeeLenaleee agree! I see that var is iused in many other samples. I think they are coming from previous version.
I fix this new ones and I'm going to submit a PR for the others